esp/src/src-react/components/DaliAdmin.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import { Pivot, PivotItem } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { pushUrl } from "../util/history";
+import { pivotItemStyle } from "../layouts/pivot";
+import { GetDFSCSV } from "./GetDFSCSV";
+import { GetDFSMap } from "./GetDFSMap";
+import { GetDFSParents } from "./GetDFSParents";
+import { GetLogicalFile } from "./GetLogicalFile";
+import { GetLogicalFilePart } from "./GetLogicalFilePart";
+import { GetProtectedList } from "./GetProtectedList";
+import { GetValue } from "./GetValue";
+import nlsHPCC from "src/nlsHPCC";
+
+interface DaliAdminProps {
+    filter?: object;
+    tab?: string;
+    name?: string;
+    baseDn?: string;
+}
+
+export const DaliAdmin: React.FunctionComponent<DaliAdminProps> = ({
+    filter,
+    tab = "getdfscsv",
+    name,
+    baseDn
+}) => {
```

Comment:
Some of these props don't appear to be used?

Reply:
unused props removed.